src/test/java/IndexerTest.java

```diff
@@ -45,7 +45,7 @@ public void testIndexer() {
                 ),
                 new TestCase(
                         new ArrayList<>(Arrays.asList(
-                                ""
+                                "    "
```

Comment:
"All tests are provided to you. Upon finish, all tests should pass. DO NOT REMOVE OR CHANGE GIVEN TESTS."